pkg/inframanager/inframanager.go

```diff
@@ -38,10 +38,19 @@ func logInit() {
 	if err != nil {
 		panic(err)
 	}
+
+	err = os.MkdirAll(logDir, 0644)
```

Comment:
why 644 and not 600? We need to avoid setting excessive permissions.

Reply:
Logs are now in manager container